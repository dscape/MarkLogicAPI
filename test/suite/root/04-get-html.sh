#!/bin/sh
curl -sH "Accept: text/html" localhost:6984