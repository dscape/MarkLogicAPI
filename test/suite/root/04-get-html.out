<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <title>MarkLogic Server Version Information</title>
  </head>
  <body>4.2-1</body>
</html>
